src/browserlib/extract-dfns.js

```diff
@@ -480,3 +484,81 @@ function preProcessHTML() {
 
     });
 }
+
+function preProcessSVG2() {
+  const idl = extractWebIdl();
+  const idlTree = parse(idl);
+  const idlInterfaces = idlTree.filter(item => item.type === "interface" || item.type === "interface mixin");
+
+  // the only element definition not properly marked up in the SVG spec
+  const linkHeading = document.getElementById("LinkElement");
+  if (linkHeading && !linkHeading.dataset.dfnType) {
+    linkHeading.dataset.dfnType = "element";
+    linkHeading.dataset.lt = "link";
+  }
+
+  [...document.querySelectorAll(".attrdef dfn[id]:not([data-dfn-type]):not([data-skip])")]
+    .forEach(el => {
+      el.dataset.dfnType = "element-attr";
+      const attrDesc = document.querySelector('[data-reffy-page$="attindex.html"] th span.attr-name a[href$="#' + el.id + '"]');
+      if (attrDesc) {
+          el.dataset.dfnFor = attrDesc.closest('tr').querySelector('td').textContent;
+      } else {
+        console.error("Could not find description for " + el.textContent);
```

Comment:
Shouldn't this (and other call below) throw instead? `console.error` messages are likely going to be ignored, e.g. drowned in other console messages produced by ReSpec.

Reply:
if we throw and the SVG spec is updated "incorrectly", what will happen? will reffy updates be blocked until this gets fixed?

(I note that in HTML spec parsing I used yet another approach of setting type "unknown"; this should be aligned in any case)